src/main/java/io/stargate/sgv2/jsonapi/api/model/command/Command.java

```diff
@@ -35,6 +36,7 @@
  *
  * <p>Each command should validate itself using the <i>javax.validation</i> framework.
  */
+@RegisterForReflection
```

Comment:
We can remove this now or?

Reply:
I will try and make sure it won't break anything but probably yes.